docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -189,279 +189,293 @@ In the YugabyteDB Anywhere UI, navigate to **Configs > Cloud Provider Configurat
-- Use **Pull Secret File** to upload the pull secret to download the image of the Enterprise YugabyteDB that is in a private repository. Your Yugabyte sales representative should have provided this secret.
+1. Specify a meaningful name for your configuration.
+2. Choose one of the following ways to specify **Kube Config** for an availability zone:
+   - Specify at **provider level** in the provider form. If specified, this configuration file is used for all availability zones in all regions.
+   - Specify at **zone level** in the region form. This is required for **multi-az** or **multi-region** deployments. If the zone is in a different Kubernetes cluster than YugabyteDB Anywhere, a zone-specific `kubeconfig` file needs to be passed.
+3. In the **Service Account** field, provide the name of the [service account](#service-account) which has necessary access to manage the cluster (see [Create cluster](../../../../deploy/kubernetes/single-zone/oss/helm-chart/#create-cluster)).
+4. In the **Image Registry** field, specify from where to pull the YugabyteDB image. Accept the default setting, unless you are hosting the registry, in which case refer to steps described in [Pull and push YugabyteDB Docker images to private container registry](../../../install-yugabyte-platform/prerequisites#pull-and-push-yugabytedb-docker-images-to-private-container-registry).
+5. Use **Pull Secret File** to upload the pull secret to download the image of the Enterprise YugabyteDB that is in a private repository. Your Yugabyte sales representative should have provided this secret.
 
 ## Configure region and zones
 
-Continue configuring your Kubernetes provider by clicking **Add region** and completing the **Add new region** dialog, as follows:
+Continue configuring your Kubernetes provider by clicking **Add region** and completing the **Add new region** dialog shown in the following illustration: 
 
-- Use the **Region** field to select the region.
+![Add new region](/images/ee/k8s-setup/k8s-az-kubeconfig.png)
 
-- Use the **Zone** field to select a zone label that should match with your failure domain zone label `failure-domain.beta.kubernetes.io/zone`.
+1. Use the **Region** field to select the region.
 
-- Optionally, use the **Storage Class** field to enter a comma-delimited value. If you do not specify this value, it would default to standard. You need to ensure that this storage class exists in your Kubernetes cluster and the following guidelines are taken into consideration:
+1. Use the **Zone** field to select a zone label that should match the value of failure domain zone label on the nodes. `topology.kubernetes.io/zone` would place the pods in that zone.
```

Comment:
> `topology.kubernetes.io/zone` would place the pods in that zone.

Actually this is the failure domain zone label on the node which is used to place the pods in a particular zone.
This sentence is not conveying that I think. It is not clear what this `topology.kubernetes.io/zone` text is.

Reply:
@bhavin192 I'm not sure how the Zone field description should be changed. Could you clarify?

Reply:
1. Use the **Zone** field to select a zone name that should match the value of failure domain zone label `topology.kubernetes.io/zone` on the nodes.

Reply:
Changed